app/views/games/play.html.slim

```diff
@@ -11,3 +11,10 @@ br
 = link_to 'Undo Round', undo_round_game_path(@game), class: 'btn btn-primary', method: 'delete'
 br
 = link_to 'Finish Game', finish_game_path(@game), class: 'btn btn-primary', method: 'patch'
+
+coffee:
+  ($ window).keypress (e) ->
+    if e.keyCode == 0 or e.keyCode == 32
+      e.preventDefault()
+      ($ "#next-button").click()
```

Comment:
The article makes sense, but for this project I would prefer to keep the coding styles as similar as possible, so I still prefer `$(selector)` over `($ selector)`